keras/src/applications/densenet.py

```diff
@@ -487,6 +487,6 @@ def decode_predictions(preds, top=5):
     A Keras model instance.
 """
 
-setattr(DenseNet121, "__doc__", DenseNet121.__doc__ + DOC)
-setattr(DenseNet169, "__doc__", DenseNet169.__doc__ + DOC)
-setattr(DenseNet201, "__doc__", DenseNet201.__doc__ + DOC)
+setattr(DenseNet121, "__doc__", (DenseNet121.__doc__ or "") + DOC)
```

Comment:
This file is using a different pattern from the other files. Please use the same pattern  with an `if`.